### DISCUSSION TITLE
Are development efforts still contiung?

### DISCUSSION THREAD

**Discussion by kaneryu** [z-huang/music#1122] [General]
I am developing a desktop app but that would require the mobile app to still be active, and it seems like it isn't currently under active development...

**Comment by z-huang** [comment]
The development is temporarily suspended because the semester has started and I have to put most of my effort on it. Can you describe more about your project and why it needs the mobile app to be active? The app is still working except some little bugs.

**Reply by kaneryu** [reply]
Well, not nessasarily, I could still make it without the mobile app being actively manintained. I would like for there to be intergration between the desktop app and the mobile app (for example, if both devices are on the same network/connceted by bluetooth, then you can share playback between them). 

My wording was incorrect in that previous post, sorry about that.

**Comment by DeadSecTr** [comment]
App Abandoned?

**Reply by kaneryu** [reply]
The developer said here 
>  The development is temporarily suspended because the semester has started and I have to put most of my effort on it.

**Comment by itsmeteddyhere** [comment]
How are you doing on this project? I've always wanted something like InnerTune on PC.

**Reply by kaneryu** [reply]
Progress is going well, but release is still far off.
I'm having to learn some new things to make it, so it will take a while

**Reply by LukeWarmpocket** [reply]
Check out Harmony Music. It has a desktop app. I've never used it, but of all the music apps I've seen it's the only one that advertised a desktop app.
